apply_weights.py

```diff
@@ -0,0 +1,134 @@
+"""
+Gregory Way 2017
+PanCancer Classifier
+apply_weights.py
+
+Usage: Run in command line with required command argument:
+
+        python apply_weights.py --classifier $summary_file
+
+Where summary_file is a string. There are also optional arguments:
+
+    --copy_number       store_true: supplement copy number to define Y
+
+Output:
+.tsv file of classifier scores and other covariate info for plotting
+"""
+
+import os
+import argparse
+import warnings
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import StandardScaler
+from tcga_util import integrage_copy_number
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-c', '--classifier',
+                    help='folder location of classifier file')
+parser.add_argument('-u', '--copy_number', action='store_true',
+                    help='Supplement Y matrix with copy number events')
+args = parser.parse_args()
+
+# Load command arguments
+classifier_file = os.path.join(args.classifier, 'classifier_summary.txt')
+copy_number = args.copy_number
+
+# Load Constants
+rnaseq_file = os.path.join('data', 'pancan_rnaseq_freeze.tsv')
+sample_file = os.path.join('data', 'sampleset_freeze_version3.csv')
+cancer_gene_file = os.path.join('data', 'vogelstein_cancergenes.tsv')
+copy_loss_file = os.path.join('data', 'copy_number_loss_status.tsv')
+copy_gain_file = os.path.join('data', 'copy_number_gain_status.tsv')
+mutation_burden = os.path.join('ddr', 'data', 'mutation-load.txt')
+mut_file = os.path.join('data', 'pancan_mutation_freeze.tsv')
+
+# Generate filenames to save output plots
+output_base_file = os.path.dirname(classifier_file)
+output_dec_file = os.path.join(output_base_file, 'classifier_decisions.tsv')
+
+# Load Data
+rnaseq_df = pd.read_table(rnaseq_file, index_col=0)
+mutation_df = pd.read_table(mut_file, index_col=0)
+sample_info = pd.read_csv(sample_file, index_col=0)
+mut_burden = pd.read_table(mutation_burden)
+
+# Summarize data based on classifier summary file
+with open(classifier_file) as class_fh:
+    for line in class_fh:
+        line = line.strip('\n').split('\t')
+        if line[0] == 'Genes:':
+            genes = line[1:]
+        if line[0] == 'Tissues:':
+            diseases = line[1:]
+        if line[0] == 'Coefficients:':
+            coef_df = pd.read_table(line[1], )
+
+# Subset Y matrix
```

Comment:
in this case, the X and Y refer to the supervised machine learning problem of the flavor `y = wx`. 

I agree that this could be confusing to a reader. I will add comments describing this and also consider renaming some of the references to y and x since not all of them are _specifically_ suitable for the above equation